### PR TITLE
Nicer icons and adding some other metadata

### PR DESCRIPTION
**Pull request by matzew**

### CODE REVIEW

File: apb.yml [hunk 1]
```diff
@@ -9,7 +9,9 @@ metadata:
   displayName: Mobile CI/CD
   longDescription: "To use the Mobile CI/CD service you must first agree to the Android SDK License Agreement. By checking this box you are agreeing to the Android Software Development Kit License agreement which can be read at the following URL: https://developer.android.com/studio/terms.html\n\n. If the box is not checked the installer will take no action."
   serviceName: aerogear-digger-apb
-  console.openshift.io/iconClass: font-icon icon-jenkins
+  imageUrl: "https://aerogear.org/img/aerogeardigger_icon_32px_cropped.png"
```
Comment (matzew) [inline apb.yml:12]: is this the best 'small' image we have online ?

Comment (matzew) [reply apb.yml:12]: @aliok  ^

Comment (david-martin) [reply apb.yml:12]: ideal size in catalog seems to be 50x50. If we could get one that size :+1:

Comment (matzew) [reply apb.yml:12]: I don't know if that size is available

**Pull request merged**